### PR TITLE
Add featured company support to spec guides

### PR DESCRIPTION
**Pull request by brandonbk**
Screen Shot 2020-09-08 at 3 32 18 PM

Screen Shot 2020-09-08 at 4 58 50 PM

### CODE REVIEW

**Review by zarathustra323 — COMMENTED** [review]
@brandonbk did you test this on FCP as well?

**Review by zarathustra323 — CHANGES_REQUESTED** [review]
Also, the build is failing:
image

you can add an ignore for the `v-html` warning, but you'll want to fix the other one

File: packages/common/browser/spec-guide/table.vue [hunk 1]
```diff
@@ -197,7 +202,10 @@ export default {
     visibleColumnList() {
       const { columnList, selectedMeasure, selectedMeasureKey } = this;
       if (!selectedMeasure) return columnList.slice();
-      const visible = columnList.filter(col => !col.measure || col.measure === selectedMeasureKey);
+      const visible = columnList.filter((col) => {
```
Comment (zarathustra323) [inline packages/common/browser/spec-guide/table.vue:205]: `const visible = columnList.filter((col) => ((!col.measure || col.measure === selectedMeasureKey) && !col.hidden);`

**Review by zarathustra323 — CHANGES_REQUESTED** [review]

**Comment by brandonbk** [comment]
@zarathustra323 - fixed lint errors and confirmed support for FCP guides.

**Review by zarathustra323 — APPROVED** [review]

**Pull request merged**